### PR TITLE
Update documentation for automatic sidecar injection webhook.

### PR DESCRIPTION
**Pull request by yusuoh**

### CODE REVIEW

File: _docs/setup/kubernetes/sidecar-injection.md [hunk 1]
```diff
@@ -139,6 +139,30 @@ Note that unlike manual injection, automatic injection occurs at the pod-level.
...
+You can also use Helm to generate the yaml file and install it manually. E.g.
+```bash
+helm template --namespace=istio-system --set sidecar-injector.enabled=true install/kubernetes/helm/istio > istio.yaml
+```
+
+```bash
+kubectl apply -f istio.yaml
+```
+
+In addtion, there are some other configuration parameters defined for sidecar
```
Comment (sdake) [inline _docs/setup/kubernetes/sidecar-injection.md]: addition is typo

File: _docs/setup/kubernetes/sidecar-injection.md [hunk 2]
```diff
@@ -139,6 +139,30 @@ Note that unlike manual injection, automatic injection occurs at the pod-level.
...
+injector webhook service in values.yaml. You can overide the default values to
```
Comment (sdake) [inline _docs/setup/kubernetes/sidecar-injection.md]: the yaml linter will probably want values.yaml in backticks.

**Comment by sdake** [comment]
I just took a look at the gate output.  May need to add override to the .spelling file.  Not sure why that isn't in the default dictionaries :)

File: _docs/setup/kubernetes/sidecar-injection.md [hunk 3]
```diff
@@ -139,6 +139,30 @@ Note that unlike manual injection, automatic injection occurs at the pod-level.
 
 #### Installing the webhook
 
+##### For version 0.8.0 and later
+
+To enable sidecar injection webhook, you can use [Helm]({{home}}/docs/setup/kubernetes/helm-install.html)
```
Comment (geeknoid) [inline _docs/setup/kubernetes/sidecar-injection.md]: to enable the sidecar

**Pull request merged**